integration/client/client_test.go

```diff
@@ -39,6 +39,12 @@ import (
 	"github.com/opencontainers/go-digest"
 	"github.com/opencontainers/image-spec/identity"
 	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"go.opentelemetry.io/otel"
+	sdktrace "go.opentelemetry.io/otel/sdk/trace"
+	"go.opentelemetry.io/otel/sdk/trace/tracetest"
```

Comment:
I think we should move the otel related setup to a separate file so that we do not need to import these packages everytime we have to use a `newInMemoryExporterTracer`.

Reply:
Have the func in another file so it can be reused without importing the same modules sounds good. 
Updated in the new commit.